.github/workflows/build.yml

```diff
@@ -0,0 +1,57 @@
+name: build
+
+on:
+  push:
+    branches:
+      - 'master'
+    tags:
+      - 'v*'
+  pull_request:
+    branches:
+      - 'master'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: |
+            tinygo/tinygo
+            ghcr.io/${{ github.repository }}/tinygo
+          tags: |
+            type=ref,event=branch
+            type=ref,event=pr
+            type=semver,pattern={{version}}
+          labels: |
+            org.opencontainers.image.title=TinyGo
+            org.opencontainers.image.description=Go compiler for small places. Microcontrollers, WebAssembly, and command-line tools. Based on LLVM.
```

Comment:
Override labels here otherwise it will take the repo name as title (`docker`) and description (`Contains the Dockerfile used to build tinygo release containers`) which seems really what you want.